test/openAPI/features/oauth_token.feature

```diff
@@ -4,24 +4,30 @@ Feature: API to create access token
     Then User receives a response from the POST /oauth/token endpoint
     And The POST /oauth/token endpoint response should be returned in a timely manner 15000ms
     And The POST /oauth/token endpoint response should have status 200
-    And The POST /oauth/token endpoint response should have content-type: application/json header
+    And The POST /oauth/token response should have "content-type": "application/json" header
     And The POST /oauth/token endpoint response should match json schema
     And The POST /oauth/token endpoint response should contain "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9" as idToken
+    And The POST /oauth/token endpoint response should contain "string.string.string" as accessToken
+    And The POST /oauth/token endpoint response should contain "Bearer" as tokenType
+    And The POST /oauth/token endpoint response should contain 0 as expiresIn
 
   @unit @positive
   Scenario Outline: The user successfully receives the id and access token
     Given The user wants to receive the id token and access token
-    When The User sends POST request with given "authorization_code" as grantType, "code" as code, "<clientId>" as clientId, "urn:ietf:params:oauth:client-assertion-type:jwt-bearer" as clientAssertionType, "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9." as clientAssertion, "<redirectUri>" as redirectUri, "<clientId>" as iss, "<clientId>" as sub, "<audience>" as aud, 1024 as exp, 1516239022 as iat
+    When The User sends POST request with given "authorization_code" as grantType, "code" as code, "<clientId>" as clientId, "urn:ietf:params:oauth:client-assertion-type:jwt-bearer" as clientAssertionType, "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9." as clientAssertion, "<redirectUri>" as redirectUri, "<clientId>" as iss, "<clientId>" as sub, "<audience>" as aud, 1024 as exp, 1516239022 as iat, "content-type": "application/x-www-form-urlencoded" header
     Then User receives a response from the POST /oauth/token endpoint
     And The POST /oauth/token endpoint response should be returned in a timely manner 15000ms
     And The POST /oauth/token endpoint response should have status 200
-    And The POST /oauth/token endpoint response should have content-type: application/json header
+    And The POST /oauth/token response should have "content-type": "application/json" header
     And The POST /oauth/token endpoint response should match json schema
     And The POST /oauth/token endpoint response should contain "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9" as idToken
+    And The POST /oauth/token endpoint response should contain "string.string.string" as accessToken
```

Comment:
Shouldn't we just check if "accessToken" is accessible?

Reply:
yeah I think so, I will change it to check for parameter only